project/Build.scala

```diff
@@ -281,9 +281,12 @@ object Build {
       }
 
       // All compiler dependencies except the library
-      val otherDependencies = dependencyClasspath.in(`dotty-compiler`, Compile).value
-        .filterNot(_.get(artifact.key).exists(_.name == "dotty-library"))
-        .map(_.data)
+      val otherDependencies = {
+        val excluded = Set("dotty-library", "dotty-compiler")
+        fullClasspath.in(`dotty-doc`, Compile).value
```

Comment:
We're still interested in the compiler dependencies. Does this work because `dotty-doc` depends on `dotty-compiler`? Maybe add a comment